### ISSUE TITLE
Please build requests-kerberos for Python 3.8

### ISSUE THREAD

**Issue by rolweber** [ContinuumIO/anaconda-issues#11918]
Hello Team Anaconda,

it is currently not possible to install `sparkmagic` with Python 3.8, because the dependency `requests-kerberos` is not available for that Python version. Could you please build that package?

`conda search "requests-kerberos>=0.12.0"`

```txt
# Name                       Version           Build  Channel             
requests-kerberos             0.12.0          py27_0  pkgs/main           
requests-kerberos             0.12.0          py35_0  pkgs/main           
requests-kerberos             0.12.0          py36_0  pkgs/main           
requests-kerberos             0.12.0          py37_0  pkgs/main           
```

Thanks and Cheers,
  Roland

**Comment by jjhelmus** [comment]
`requests-kerberos` packages are now available for Python 3.8:

```
~$ conda search requests-kerberos=0.12.0
Loading channels: done
# Name                       Version           Build  Channel             
requests-kerberos             0.12.0          py27_0  pkgs/main           
requests-kerberos             0.12.0          py35_0  pkgs/main           
requests-kerberos             0.12.0          py36_0  pkgs/main           
requests-kerberos             0.12.0          py37_0  pkgs/main           
requests-kerberos             0.12.0          py38_0  pkgs/main       
```

**Comment by rolweber** [comment]
Thanks a lot!
